### PR TITLE
Refactoring the names of the variable for clarity and consistency in discrete_log.ipynb 

### PR DESCRIPTION
**Pull request by umer066**
Updated the variable names for improved the readability and adherence

### CODE REVIEW

File: algorithms/algebraic/discrete_log/discrete_log.ipynb [hunk 1]
```diff
@@ -85,20 +85,20 @@
     "\n",
     "@qfunc\n",
     "def discrete_log_oracle(\n",
-    "    g: CInt,\n",
-    "    x: CInt,\n",
-    "    N: CInt,\n",
+    "    g_generator: CInt,\n",
+    "    x_logarithm: CInt,\n",
```
Comment (orsa-classiq) [inline algorithms/algebraic/discrete_log/discrete_log.ipynb:89]: Notice that x is not the logarithm, but a group element.
I suggest `x_element` instead

**Review by orsa-classiq — CHANGES_REQUESTED** [review]
Thanks for contributing! see my comment please

**Comment by orsa-classiq** [comment]
fixing #42

File: algorithms/algebraic/discrete_log/discrete_log.ipynb (hunk 1 above)
Comment (umer066) [reply algorithms/algebraic/discrete_log/discrete_log.ipynb:89]: Thank You for your suggestion. I will update it soon

**Review by orsa-classiq — APPROVED** [review]

**Pull request merged**